### DISCUSSION TITLE
How to install version 3.6 on Ventura?

### DISCUSSION THREAD

**Discussion by mkonia** [Homebrew/discussions#3623] [Everyday usage]
How to install version 3.6 on Ventura 13.0 Beta (22A5331f)? When I enter the usual installation command I get this message: Can’t install the software [Command Line Tools] because it is not currently available from the Software Update server.

**Comment by SMillerDev** [comment]
You'll have to figure out how to install the command line tools on your own. We don't support prerelease software.
